src/pages/components/externalPages/TestimonialExternalPages/TestimonialHeaderContent.js

```diff
@@ -10,7 +10,7 @@ const TestimonialHeaderContent = ({ mode, item }) => (
       alt={item.name}
     />
     <figcaption className='absolute left-0 right-0 bottom-5 h-full bg-gradient-to-r from-orange-900 dark:from-dark-ash-900'>
-      <div className='px-[20px] pt-[64px] sm:px-[55px] sm:pt-[38px] lg:h-[572px] lg:px-[100px] lg:pt-[51px] xl:px-[200px] 2xl:px-[339px]'>
+      <div className='h-[710px] px-5 pt-[4.8rem] sm:px-8 md:px-14 lg:h-[572px] xl:px-[200px] 2xl:px-[240px] 3xl:px-[320px]'>
         <Link to='/'>
           <div className='h-[39px] w-[165px] hover:cursor-pointer sm:m-auto lg:h-[48px] lg:w-[192px]'>
             <LazyLoadImage
```

Comment:
# Comment 

- [ ] In TestimonialHeaderContent please configure the h1 tag change the lg:mt-[77px] to lg:mt-[45px] and and xl:mt-[77px].

# Capture
![Capture](https://user-images.githubusercontent.com/89382909/169990936-be9c97d0-86fe-4cb9-a792-0ca8051699bf.PNG)